scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/core/properties/PASchedulerProperties.java

```diff
@@ -393,6 +393,9 @@ negative number indicates that session is infinite (value specified in second) *
 
     LICENSE_SCHEDULING_POLICY_CONFIGURATION("pa.scheduler.license.policy.configuration", PropertyType.STRING),
 
+    /** location of the jdbm database for persistent license registrations */
+    LICENSE_SCHEDULING_POLICY_DATABASE("pa.scheduler.license.policy.db", PropertyType.STRING, "data/licenses"),
```

Comment:
please add the value in the config/scheduler/settings.ini file as well

Reply:
Forgot to commit settings.ini. Done.